curvefs/proto/topology.proto

```diff
@@ -351,6 +352,14 @@ message CreatePartitionResponse {
     repeated common.PartitionInfo partitionInfoList = 2;
 }
 
+message DeletePartitionRequest {
+    required uint32 partitionId = 1;
+}
+
+message DeletePartitionResponse {
```

Comment:
Can you give some comments to indicate the statusCode that may be returned, and the module (or reason) that produces a non-ok statusCode, so that people who use this interface can be guided to deal with this information.

Reply:
The different statusCode represent different meanings, for example, CopysetNotFound, DeleteFailOnMetaserver and so on.